migrations/versions/201506051421_4074211727ba_add_indico_unaccent_function.py

```diff
@@ -0,0 +1,37 @@
+"""ADD indico_unaccent function
+
+Revision ID: 4074211727ba
+Revises: 3f3a9554a6da
+Create Date: 2015-06-05 14:21:52.752777
+"""
+
+from indico.core.db.sqlalchemy.core import db
+
+# revision identifiers, used by Alembic.
+revision = '4074211727ba'
+down_revision = '3f3a9554a6da'
+
+
+def upgrade():
+    db.session.execute('''
+        CREATE OR REPLACE FUNCTION indico_unaccent(value TEXT)
+            RETURNS TEXT
+        AS $$
+        BEGIN
+           IF EXISTS(SELECT TRUE FROM pg_extension WHERE extname = 'unaccent')
```

Comment:
Do we really want this check at runtime every single time the function is called? I'd rather perform the check in the migration script and then create one or another function depending on the result.

You can use `conn = op.get_bind()` to get the DB connection in a way that lets you perform SELECT queries. See e.g. this example on how to use it: https://github.com/indico/indico/blob/master/migrations/versions/201505180930_222b9d8d4e22_use_acl_mixin_for_blocking_principals.py